MaxText/layers/embeddings.py

```diff
@@ -56,7 +56,7 @@ def setup(self):
         'embedding',
         with_logical_partitioning(self.embedding_init, ('vocab', 'embed')),
         (self.num_embeddings, self.features),
-        jnp.float32,
```

Comment:
I gave feedback by chat but please use a config other than config.dtype (for example config.weight_dtype would be good)